docs/ref/contrib/gis/tutorial.txt

```diff
@@ -746,7 +746,7 @@ position.
 ~~~~~~~~~~~~~~~
 
 With the :class:`~django.contrib.gis.admin.OSMGeoAdmin`, GeoDjango uses
-a `Open Street Map`_ layer in the admin.
+an `OpenStreetMap`_ layer in the admin.
```

Comment:
This a link, see line 739, which will need updating.

Reply:
Thanks, @smithdc1. I have now fixed it.